static/js/zamboni/validator.js

```diff
@@ -253,7 +253,7 @@ function initValidator($doc) {
             }
 
             if (this.fileURL) {
-                var url = this.fileURL + file;
+                var url = this.fileURL + '?path=' + file;
```

Comment:
it's lucky this works.  I assume you've tested it still works?  (`file` is the same as code manage expects)

Reply:
Yes, I tested it manually and it seems to work. We actually have a Python function that will build this URL for us, including the `path` param, but this is JS, and the `file` value is only available at this point where we're processing the json file, so a Python function doesn't help us at this point.

I could have abstracted this into a function, but I figured the less I touched the JS code the better! 😉